packages/api/src/directive/controller.test.ts

```diff
@@ -195,7 +194,7 @@ describe("POST /directive", () => {
 			.expect(200);
 
 		const directiveResult = await db.query<Directive>(
-			`SELECT 
+			`SELECT
```

Comment:
(We should definitely set up prettier in here and just run it once and for all!)